app/_locales/en/messages.json

```diff
@@ -3115,6 +3115,12 @@
   "notificationsPortfolioV2Title": {
     "message": "MetaMask Portfolio - join millions"
   },
+  "notificationsSimulationsDescriptionOne": {
+    "message": "Now you can see the potential outcome of your transactions before you make them!"
+  },
+  "notificationsSimulationsDescriptionTwo": {
+    "message": "This is just a simulation, so we can’t guarantee the final outcome. You can turn this off anytime in \nSettings > Security & Privacy. "
```

Comment:
Is `\n` necessary before `Settings > Security`?

Reply:
We did it intentionally to make that route visible in the new line.

https://github.com/MetaMask/metamask-extension/pull/23604#discussion_r1532799079